husky_components/datagrid/decorators/table-view.js

```diff
@@ -442,7 +444,7 @@ define(function() {
             if (!!this.options.selectItem && !!this.options.selectItem.type) {
                 var $cell = this.sandbox.dom.createElement(templates.headerCell);
                 this.sandbox.dom.addClass($cell, constants.checkboxCellClass);
-                if (this.options.selectItem.type === selectItems.CHECKBOX) {
+                if (this.options.selectItem.type === selectItems.CHECKBOX && this.options.disabledItems.length === 0) {
```

Comment:
i have introduced this feature but i don't need it in the master PR i will extract it into an own PR which can be discussed separately.